libraries/Microsoft.Bot.Connector/Authentication/Authenticators.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't understand why adding Arlington as a new auth point requires a new class called "Authenticators". CAn you elaborte with a nice long <summary></summary> comment on the class?